e2e/src/usecases/HomeFeed.js

```diff
@@ -0,0 +1,41 @@
+import jestExpect from 'expect'
+import { waitForElementId } from '../utils/utils'
+import { sleep } from '../../../src/utils/sleep'
+
+export default HomeFeed = () => {
+  it('should show correct information on tap of feed item', async () => {
+    // Load Wallet Home
+    await waitForElementId('WalletHome')
+    const items = await element(by.id('TransferFeedItem')).getAttributes()
+
+    // Tap top TransferFeedItem
+    await element(by.id('TransferFeedItem')).atIndex(0).tap()
+
+    // Assert on text based on elements returned earlier
+    const address = items.elements[0].label.split(' ')[0]
+    await expect(element(by.text(address)).atIndex(0)).toBeVisible()
+
+    // TODO: enable after display is standardized between feed and transaction details
```

Comment:
is there a ticket this is blocked on?

Reply:
Existing issue that was uncovered in this work. I'll make bug ticket.